index.php

```diff
@@ -102,6 +117,16 @@
 }
 
 if ($helper->authenticate()) {
+    if ($helper->is_user_already_loggedin()) {
+        //Setting loginas cookie
+        $context = context_system::instance();
+        $genratecookie = has_capability('local/signin:manage', $context);
```

Comment:
Can we change this to the correct spelling, should be "generatecookie".

Reply:
Implemented the suggested changes.